lib/contracts/flowdock-flow.ts

```diff
@@ -0,0 +1,25 @@
+import type { ContractDefinition } from 'autumndb';
+
+export const flowdockFlow: ContractDefinition = {
+	slug: 'flowdock-flow',
+	type: 'type@1.0.0',
+	name: 'Flowdock flow',
+	markers: [],
+	data: {
+		schema: {
+			required: ['data'],
+			type: 'object',
+			properties: {
+				data: {
+					required: ['name'],
```

Comment:
There is a root level `name` field so don't need to duplicate it with a `data.name` field.

Reply:
great